GRADUATION: Write up the Governance and Steering documents (#6260)
cert-manager/cert-manager: STEERING.md

> # Linkerd Steering Committee Charter

Reviewer: ```suggestion
# Steering Committee Charter
```

meets the needs of its current and future users. The Steering Committee
represents the voice of the user, and works with the cert-manager maintainers to
help guide development efforts towards solving concrete and immediate problems
for cert-manager adopters.

The members of the Steering Committee are:

- TBD

## Steering Committee Responsibilities

The Steering Committee’s responsibilities are to:

1. Provide feedback to project maintainers about cert-manager's featureset, UX,
   and operational model.
2. Assist cert-manager maintainers in prioritizing upcoming roadmap items and
   planned work.
3. Represent the "voice of the cert-manager user" both internally and to the
   broader community.
4. Provide neutral mediation for non-technical disputes.
5. Develop and maintain a project continuity plan.

## Steering Committee Membership

The Steering Committee comprises at most 7 people. To be eligible for membership
in the Steering Committee, you must:

1. Be responsible for a production cert-manager deployment of non-trivial size
2. Be willing and able to attend regularly-scheduled Steering Committee meetings
3. Abide by cert-manager's Code of Conduct.

> Candidates for membership will be nominated by current Steering Committee
> members or by cert-manager maintainers. If there are more nominations than
> Steering Committee seats remaining, the same group shall vote on the candidates,
> and the candidates with the most votes will become Steering Committee members.
> Any ties will be broken by cert-manager maintainers.

Reviewer 2: suggestion: I got lost a bit reading this and I think some of it might benefit from being clearer about the process.

How does this work? Does it match your intentions? I don't mind much about the actual process - I'm just trying to ensure that it's clear what happens in each situation!

```suggestion
Candidates for membership will be nominated by current Steering Committee
members or by cert-manager maintainers. To be accepted as a candidate, a
person must have at least two nominators from this group.

If there are fewer proposed candidates than there are open seats, then the proposed
candidates may assume their seats immediately.

If there are more nominations than there are open seats remaining, existing
Steering Committee members may vote for candidates using the
[single transferable vote](https://en.wikipedia.org/wiki/Single_transferable_vote)
system wherein candidates are ranked in order.

In the event of a tie, the tie shall be broken by a simple vote among cert-manager maintainers.
If the vote among maintainers is tied, then the tie shall be broken by random chance using a method
to be determined fair by all participants.
```

Author: I admit that I copy-pasted that paragraph from Linkerd's [STEERING charter](https://github.com/linkerd/linkerd2/blob/main/STEERING.md). I find your version better: assuming the seat immediately makes sense, and the precisions around the "tie" are good.

> Membership expires if any of the eligibility conditions is unmet, or after one
> year. Members may seek reinstatement immediately in accordance with the rules
> above.

Reviewer 2: suggestion:

```suggestion
Membership expires if any of the eligibility conditions are unmet, or after one
year. Members may seek reinstatement immediately in accordance with the rules
above.
```

## Steering Committee Meetings

> Meetings will happen periodically not less than once a quarter. Recordings and
> minutes will be posted publicly.

Reviewer 2: suggestion:

```suggestion
Steering Committee meetings will happen at least once per quarter but can be more frequent if needed.

All recordings and minutes will be posted publicly.

It is the responsibility of the Steering Committee to ensure that meetings are scheduled and run.
```

> ## Changes to this charter

Reviewer 2: nitpick (non-blocking): Every other title is title cased except this one!

```suggestion
## Changes to This Charter
```
